lib/src/common/managed_stream_subscription.dart

```diff
@@ -19,7 +19,7 @@ class ManagedStreamSubscription<T> implements StreamSubscription<T> {
   Completer<Null> _didComplete = new Completer();
 
   ManagedStreamSubscription(Stream<T> stream, void onData(T),
-      {void onError(error, [stackTrace]), void onDone(), bool cancelOnError})
+      {Function onError, void onDone(), bool cancelOnError})
```

Comment:
^ Yup, that is correct. Also, this `ManagedStreamSubscription` class is only an internal helper. The public `listenToStream()` method was already properly typing the `onError` callback param as just `Function`.